panic on `skywire cli tp add`

```
go run github.com/skycoin/skywire/cmd/skywire@develop cli tp add $_pk
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x70 pc=0x17e4975]

goroutine 1 [running, locked to thread]:
github.com/skycoin/skywire/cmd/skywire-cli/commands/tp.PrintTransports(0xc000299500, {0xc000116070, 0x1, 0x1})
	/home/skycoin/go/pkg/mod/github.com/skycoin/skywire@v1.3.29-rc7.0.20250313050427-65ff378bfe94/cmd/skywire-cli/commands/tp/tp.go:335 +0x195
github.com/skycoin/skywire/cmd/skywire-cli/commands/tp.init.func1(0x324cde0, {0xc00004d040, 0x1, 0x1c16c69?})
	/home/skycoin/go/pkg/mod/github.com/skycoin/skywire@v1.3.29-rc7.0.20250313050427-65ff378bfe94/cmd/skywire-cli/commands/tp/tp.go:282 +0xaf9
github.com/spf13/cobra.(*Command).execute(0x324cde0, {0xc00004d010, 0x1, 0x1})
	/home/skycoin/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:989 +0xa91
github.com/spf13/cobra.(*Command).ExecuteC(0x323d6a0)
	/home/skycoin/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1117 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
	/home/skycoin/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1041
github.com/skycoin/skywire/cmd/skywire/commands.Execute()
	/home/skycoin/go/pkg/mod/github.com/skycoin/skywire@v1.3.29-rc7.0.20250313050427-65ff378bfe94/cmd/skywire/commands/root.go:369 +0x1a
main.main()
	/home/skycoin/go/pkg/mod/github.com/skycoin/skywire@v1.3.29-rc7.0.20250313050427-65ff378bfe94/cmd/skywire/skywire.go:35 +0x59
exit status 2

```

--- a/.gitignore
+++ b/.gitignore
@@ -17,6 +17,8 @@ tmp/
 build/
 ./apps/
 ./skywire/
+./local
+local/
 ./local/*
 ./local*/*
 ./transport_logs
@@ -28,6 +30,7 @@ rewards
 ./pkg/visor/bar/
 ./pkg/visor/foo/
 
+*.json
 ./bin
 ./node
 ./users.db

--- a/cmd/skywire-cli/commands/tp/tp.go
+++ b/cmd/skywire-cli/commands/tp/tp.go
@@ -329,8 +329,10 @@ func PrintTransports(cmdFlags *pflag.FlagSet, tps ...*visor.TransportSummary) {
 	}
 
 	var outputTPS []outputTP
-
 	for _, tp := range tps {
+		if tp == nil {
+			continue
+		}
 		tpMode := "regular"
 		if tp.IsSetup {
 			tpMode = "setup"
@@ -347,6 +349,7 @@ func PrintTransports(cmdFlags *pflag.FlagSet, tps ...*visor.TransportSummary) {
 
 		_, err = fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\n", tp.Type, tp.ID, tp.Remote, tpMode, tp.Label)
 		internal.Catch(cmdFlags, err)
+
 	}
 	internal.Catch(cmdFlags, w.Flush())
 	internal.PrintOutput(cmdFlags, outputTPS, b.String())

